client-cli/src/command/address_command.rs

```diff
@@ -14,7 +14,9 @@ const ADDRESS_TYPE_VARIANTS: [&str; 2] = ["transfer", "staking"];
 #[derive(Debug)]
 pub enum AddressType {
     Transfer,
+    TransferWatch,
     Staking,
+    StakingWatch,
```

Comment:
I think no need for this, as it's public information -- just use "state" command to retrieve it: https://github.com/crypto-com/chain/pull/1035

Reply:
Ok